### ISSUE TITLE
Please update the exe from "Chaturbate.com" to "Chaturbate.global" as ".com" domain blocked in many places

### ISSUE THREAD

**Issue by fffantasies** [teacat/chaturbate-dvr#74]

**Comment by J0nDoe** [comment]
@fffantasies this should be resolved since the changes have been merged in :-D

**Comment by YamiOdymel** [comment]
Yes, thanks to @J0nDoe, this has been merged into [v2.0.0](https://github.com/teacat/chaturbate-dvr/releases/tag/v2.0.0).

It's now possible to change the domain using the `-domain` argument, with `https://chaturbate.global/` set as the default value.
